ajax/libs/bootstrap-confirmation/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "bootstrap-confirmation",
+  "filename": "bootstrap-confirmation.min.js",
+  "version": "1.0.2",
+  "description": "Confirmation plugin compatible with Twitter Bootstrap 3 extending Popover",
+  "homepage": "https://github.com/tavicu/bs-confirmation",
+  "keywords": [
+    "bootstrap",
+    "confirm",
+    "confirmation"
+  ],
+  "maintainers": [
+    {
+      "name": "Octavian Ruda",
+      "web": "https://github.com/tavicu"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/tavicu/bs-confirmation.git"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tavicu/bs-confirmation.git",
+    "basePath": "/",
+    "files": [
+      "*.js"
```

Comment:
@PeterDaveHello As the library is just one file, should I then just put that one under "files"?